docs/using_pyodide_from_javascript.md

```diff
@@ -64,4 +64,58 @@ pyodide.loadPackage('matplotlib').then(() => {
 
 ## Complete example
 
-TODO
+Grab the main pyodide.js and supporting files referenced in the demo project (https://alpha.iodide.io/notebooks/300/)
+
+```bash
+mkdir pyodide_local ; cd pyodide_local
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.js
+curl -O https://alpha.iodide.app/pyodide-0.10.0/packages.json
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.js
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.wasm
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.data
+curl -O https://alpha.iodide.app/pyodide-0.10.0/pyodide.asm.data.js
+```
```

Comment:
It would be better to grab a release tarball from `https://github.com/iodide-project/pyodide/releases/`.  This will have all the available packages in it.

Reply:
updated